### DISCUSSION TITLE
Filter out results in 'GET_BRAND_ANALYTICS_SEARCH_TERMS_REPORT' using additional parameters in API endpoint?

### DISCUSSION THREAD

**Discussion by ykuni-software-developer** [amzn/selling-partner-api-models#3639] [General]
Hello,
I have tried getting the data for 'GET_BRAND_ANALYTICS_SEARCH_TERMS_REPORT' using the report Api endpoint. It returns a large number of results for the week period. We only want to get relevant results based on ASIN or keywords used. I was just wondering whether it is possible to only return the results based on ASIN or keywords?

**Comment by HeGuanyuan** [comment]
I completely agree with this request. The current search terms report returns an overwhelming amount of data, especially when we need to retrieve and save the data daily or weekly. It would be extremely helpful to have the ability to filter the results based on ASIN or search terms to increase the relevance and usefulness of the data.
